src/test/regress/expected/failure_savepoints_0.out

```diff
@@ -0,0 +1,348 @@
+SELECT citus.mitmproxy('conn.allow()');
```

Comment:
I typically add comment to the sql file explaining why we have two output files. Given that this is a new case, it'd be nice to have such a comment.

Reply:
done.